### PR TITLE
observe cloud provider from config api

### PR DESCRIPTION
**Pull request by sanchezl**
Get cloud provider from `infrastructures.config.openshift.io/cluster` instead of cluster `cm/cluster-config-v1.kube-system`.

### CODE REVIEW

File: pkg/operator/starter.go [hunk 1]
```diff
@@ -79,10 +80,12 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 	if err != nil {
 		return err
 	}
+	configInformers := configinformers.NewSharedInformerFactory(configClient, 10*time.Minute)
```
Comment (deads2k) [inline pkg/operator/starter.go]: move to where other clients and informers are created.

don't forget to start it.

Comment (sanchezl) [reply pkg/operator/starter.go]: Done.

**Comment by deads2k** [comment]
known flake

**Pull request merged**